src/parser/csql_grammar.y

```diff
@@ -28434,6 +28434,67 @@ pt_fill_conn_info_container(PARSER_CONTEXT *parser,  int buffer_pos, container_1
     ctn->c10 = FROM_NUMBER(set_bits);
 }
 
+static bool
+pt_check_one_stmt(char* p)
+{
+    char end;
+    char* t = strchr(p, ';');
+    if(!t)
+     {
+        return true;
+     }
+
+    t++;
+    while (*t == ' ' || *t == '\t' || *t == '\r' || *t == '\n')
+     {
+        t++;
+     }
+    if(*t == '\0')
+       return true;
+
+    while(*p)
+    {
+        if( *p == '[' || *p == '"' || *p == '`' || *p == '\'')
+          {
+                end = (*p == '[') ? ']' : *p;
+                for(p++; *p ; p++)
+                {
+                    if (*p == end)
+                    {
+                       p++;
+                       break;
+                    }
+                }
+          }
+        else if(*p == ';')
+          {
+             p++;
+
+             /* Notice;
+             **   Forces not to have multiple statements.
+             **   Because of this processing, this function will in fact always return true.
+             **   When multiple statements come,
+             **   it is a policy that forces only the first statement to be executed without error handling.
+             **   For this, a single line of code is added below: "*p = '\0'".
+             */
+             *p = '\0';
+
+             break;
+          }
+        else
+          {
+            p++;
+          }
+    }
+
+    while (*p == ' ' || *p == '\t' || *p == '\r' || *p == '\n')
+     {
+        p++;
+     }
+
+    return (*p == '\0');
```

Comment:
If there is another statement after the second semicolon, this return statement returns false, which is not in accordance with the above notice.

Reply:
The second semicolon does not meet.
Because I forcefully inserted '\0' after the first semicolon.

Reply:
Yes, you are right. I misunderstood it.